plugin/src/test/kotlin/tanvd/grazi/grammar/LangDetectorTest.kt

```diff
@@ -0,0 +1,202 @@
+package tanvd.grazi.grammar
+
+import tanvd.grazi.GraziTestBase
+import tanvd.grazi.language.Lang
+import tanvd.grazi.language.LangDetector
+
+class LangDetectorTest : GraziTestBase(true) {
```

Comment:
Let's think about adding such a test for markdown. To check that higlighting also works for all languages.

Reply:
Added